public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -0,0 +1,197 @@
+  const inlineEditKey = 'inlineEditPanel';
+
+  const defaultMeasurements = { width: 350, height: 400 };
+  const defaultX = btnInlineEdit.x - btnInlineEdit.width + OFFSET;
+  const defaultY = -OFFSET - defaultMeasurements.height;
+
+  const savedPlacement = store.getObject(inlineEditKey, {
+    x: defaultX,
+    y: defaultY,
+    w: defaultMeasurements.width,
+    h: defaultMeasurements.height,
+  });
+  const [measurements, setMeasurements] = useState<Dimensions2D>({ width: savedPlacement.w, height: savedPlacement.h });
+  const [placement, setPlacement] = useState({ x: savedPlacement.x, y: savedPlacement.y });
+
+  const pane = useMemo(() => {
+    const state: InstanceState = instanceState;
+    if (!state) {
+      return new OptionsPaneCategoryDescriptor({ id: 'root', title: 'root' });
+    }
+
+    const supplier = (builder: PanelOptionsEditorBuilder<any>, context: StandardEditorContext<any>) => {
+      builder.addNestedOptions(getLayerEditor(instanceState));
+
+      const selection = state.selected;
+      if (selection?.length === 1) {
+        const element = selection[0];
+        if (!(element instanceof GroupState)) {
+          builder.addNestedOptions(
+            getElementEditor({
+              category: [`Selected element (${element.options.name})`],
+              element,
+              scene: state.scene,
+            })
+          );
+        }
+      }
+    };
+
+    return getOptionsPaneCategoryDescriptor({}, supplier);
+  }, [instanceState]);
+
+  const onDragStop = (event: any, dragElement: any) => {
+    setPlacement({ x: dragElement.x, y: dragElement.y });
+    saveToStore(dragElement.x, dragElement.y, measurements.width, measurements.height);
+  };
+
+  const onResizeStop = (event: SyntheticEvent<Element, Event>, data: ResizeCallbackData) => {
+    const { size } = data;
+    setMeasurements({ width: size.width, height: size.height });
+    saveToStore(placement.x, placement.y, size.width, size.height);
+  };
+
+  const saveToStore = (x: number, y: number, width: number, height: number) => {
+    store.setObject(inlineEditKey, { x: x, y: y, w: width, h: height });
+  };
+  return (
+    <Draggable handle="strong" onStop={onDragStop} position={{ x: placement.x, y: savedPlacement.y }}>
+      <Resizable height={measurements.height} width={measurements.width} onResize={onResizeStop}>
+        <div
+          className={cx('box', 'no-cursor', `${styles.inlineEditorContainer}`)}
+          style={{ height: `${measurements.height}px`, width: `${measurements.width}px` }}
+          ref={ref}
+        >
+          <strong className={cx('cursor', `${styles.inlineEditorHeader}`)}>
+            <div className={styles.placeholder} />
+            <div>Canvas Inline Editor</div>
+            <IconButton name="times" size="xl" className={styles.inlineEditorClose} onClick={onClose} />
+          </strong>
+          <div style={{ overflow: 'scroll' }} className={styles.inlineEditorContentWrapper}>
```

Comment:
should `overflow` be part of the class?

Reply:
yes, overflow is part of the class. Forgot to remove the inline style.